src/ApparentHorizons/ApparentHorizonComputeItems.hpp

```diff
@@ -0,0 +1,70 @@
+// Distributed under the MIT License.
```

Comment:
Please rename the file `ComputeItems.hpp`, its in the ApparentHorizons directory so we don't need ApparentHorizon twice.